Sources/Tracking/autogenerated/AutoDependencyInjection.generated.swift

```diff
@@ -74,6 +77,18 @@ extension DIGraph {
         CioCleanupRepository(queue: queue)
     }
 
+    // DeviceAttributesProvider
+    var deviceAttributesProvider: DeviceAttributesProvider {
+        if let overridenDep = overrides[String(describing: DeviceAttributesProvider.self)] {
+            return overridenDep as! DeviceAttributesProvider
+        }
+        return newDeviceAttributesProvider
+    }
+
+    private var newDeviceAttributesProvider: DeviceAttributesProvider {
```

Comment:
How about renaming to something meaningful or add a comment ?

Reply:
This file is automatically generated. `newX` property is meant to mean "new instance of X". 

If you look below, all dependencies in this file are ~~written~~ generated in the same way